health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/index.js

```
@@ -136,13 +141,13 @@ const App = ({ path, stateCode, userType, tenants,BOUNDARY_HIERARCHY_TYPE, hiera
       },
     },
     config: {
-      cacheTime:Infinity,
+      cacheTime: Infinity,
```

Biome

[error] Use Number.Infinity instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.Infinity instead.


(lint/style/useNumberNamespace)